.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add "Fixes #XXX", replacing XXX with the issue number.
+
+Be sure that your contribution follows the [Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/).
+
+Be sure to go over each item in the list below before submitting your pull request.
+-->
+
+### Description
+
+Briefly describe what your changes are doing. It is more important to mention why the changes are necessary, rather than the how.
+
+### Checklist
+
+- [ ] The commit messages include a [DCO](https://discourse.ros.org/t/starting-to-enforce-developer-certificate-of-origin-dco-for-some-ros-2-repos/7420).
+- [ ] The commit messages follow [good practices](https://chris.beams.io/posts/git-commit/).
+- [ ] Tests have been added or updated to cover the changes. If tests cannot be added for some reason, it has been explained in the description.
+- [ ] The changes passes tests locally (see [here](https://index.ros.org/doc/ros2/Tutorials/Colcon-Tutorial) for how to run tests.
+- [ ] The PR is minimal and any unrelated changes are submitted separately.
+- [ ] The API documentation has been updated based on changes in APIs or addition of new APIs, if any.
+- [ ] The [documentation](https://index.ros.org/doc/ros2/) has been updated to describe the new feature or package added, if any.
```

Comment:
For many kind of changes https://index.ros.org/doc/ros2/ isn't relevant. So this might confuse a contributor.

Reply:
Should we eliminate this or provide something else along with it?